### PR TITLE
Update ownerless-groups-teams.md

### PR DESCRIPTION
**Pull request by MikePark-N**

### CODE REVIEW

File: microsoft-365/admin/create-groups/ownerless-groups-teams.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ To set an ownerless group or team policy
 1. Type the number of active members that you want to notify and select the number of weeks to send the notification. (The notification list is created during the first notification and does not change.) Select **Next**.
 
 1. On the *Who is this email coming from* page, select a sender for the email, and then select **Next**.
+> [!Note]
+> Shared mailboxes are not supported. The sender must be either a User Mailbox or a Group Mailbox.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/ownerless-groups-teams.md:55]:
```suggestion
> Shared mailboxes are not supported. The sender must be either a user mailbox or a group mailbox.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MikePark-N, kindly commit the minor suggested changes. Many thanks.

File: microsoft-365/admin/create-groups/ownerless-groups-teams.md (hunk 1 above)
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/ownerless-groups-teams.md:55]:
```suggestion

> [!NOTE]
> Shared mailboxes are not supported. The sender must be either a user mailbox or a group mailbox.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
@MikePark-N, I notice that you have marked my suggested changes as resolved, but you did not actually commit them. Could you please do so? Thanks.

**Review by MikePark-N — COMMENTED** [review]
commit.

**Review by MikePark-N — COMMENTED** [review]
👍

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by MikePark-N** [comment]
Done.

**Pull request closed without merging**